src/components/Button/BugButton/BugButton.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { BugButton } from './BugButton';
+import { expect, vi, test, describe, beforeEach } from 'vitest';
+import { BrowserRouter } from 'react-router-dom';
+import ErrorBoundary from '../../ErrorBoundary/ErrorBoundary';
+
+describe('BugButton', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  console.log = vi.fn();
+  console.error = vi.fn();
```

Comment:
Do you need consol.log here?

Reply:
Yes, I need this to avoid questions during the review from the reviewer 1. Someone may get points due to test error in console :/